packages/parcel-transformer/src/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { Transformer } from '@parcel/plugin';
+import semver from 'semver';
+
+/**
+ * Compiled Transformer
+ *
+ * Should run before any other Babel transformers.
+ */
+export default new Transformer({
```

Comment:
Not much code needed for transformer? 🤔 If it is this small, then noice 😄

Reply:
Would be a bit more involved to pick out the dependencies that we need to "include" - but yeah

Reply:
Would it be better if we follow a pattern like `*.compiled.ts` and then glob those files?

Reply:
Out of the box we won't recommend that (because of CSS prop) - but we'll definitely document that as a performance optimization your build tooling could opt into.

Reply:
Also for context when I say dependencies it's talking about files that are statically evaluated and included in the "importing" component - which #344 goes into a bit of detail around why the DX is a bit painful ATM.